lib/matplotlib/image.py

```diff
@@ -1421,9 +1421,16 @@ def imsave(fname, arr, vmin=None, vmax=None, cmap=None, format=None,
 def pil_to_array(pilImage):
     """Load a PIL image and return it as a numpy array.
 
-    Grayscale images are returned as ``(M, N)`` arrays.  RGB images are
-    returned as ``(M, N, 3)`` arrays.  RGBA images are returned as ``(M, N,
-    4)`` arrays.
+    Returns
```

Comment:
Anything to say about the input here? What is `pilImage`? (Just mentionning because this kind of numpydoc style looks strange with only `Returns` but no `Parameters`)

Reply:
Well, the input is a PIL image instance. Not much more to say. I've added a link.